client/src/app.jsx

```diff
@@ -1,21 +1,70 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
+import Search from './Search.jsx';
+import Businesshours from './Businesshours.jsx';
+import Businessinfo from './Businessinfo.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-    }
+      businessHours: {},
+      businessInfo: {},
+      address: [],
+    };
+  }
+
+  componentDidMount() {
+    this.getBusinessInfo(1);  
+  }
+
+  getBusinessInfo(ID) {
+    $.ajax({
+      type: 'GET',
+      url: `/businesses/${ID}/business_info`,
+      contentType: 'application/json',
+      success: (data) => {
+        const businessInfo = {};
+        Object.entries(data[0]).forEach(([key, value]) => {
+          if (value === 0) {
+            businessInfo[key] = 'No';
+          } else if (value === 1) {
+            businessInfo[key] = 'Yes';
+          }
+        });
+        const businessHours = {};
+        Object.entries(data[0]).forEach(([key, value]) => {
+          if (key === 'mon' || key === 'tue' || key === 'wed' || key === 'thu' || key === 'fri' || key === 'sat' || key === 'sun') {
+            businessHours[key] = value;
+          }
+        });
+        console.log(businessInfo)
+        this.setState({
+          businessHours: businessHours,
+          businessInfo: businessInfo,
+        });
+      },
+    });
+  }
+
+  // getBusinessAddress() {
+
+  // }
+
+  handleSearch(searchValue) {
   }
   
-  render () {
+  render() {
     return (
       <div>
-      STUFF WORKS
+        <Search handleSearch={this.handleSearch.bind(this)} />
```

Comment:
The linter might be giving you read because you're using .bind in the render component. It is apparently better for performance to write the binding inside the constructor, immediately below state:
  constructor(props) {
  super(props)
  this.state = { }
  this.handleSearch = this.handleSearch.bind(this)
  }

then in the render function, just write:
 <Search handleSearch={this.handleSearch} />

Your linter will be happy.

Reply:
onChange={this.handleSearch}